v-next/hardhat/test/fixture-projects/cli/telemetry/analytics/analytics-subprocess.js

```diff
@@ -0,0 +1,22 @@
+// This file will be copied to the "analytics" folder so it will be executed as a subprocess.
+// This will allow checking that the subprocess received the correct payload.
+
+// We need to directly use the node method "fs" instead of "writeJsonFile" because a TypeScript ESM file cannot be imported without compiling it first
+import { writeFileSync } from "node:fs";
+import path from "node:path";
+
+const PATH_TO_RESULT_FILE = path.join(
+  process.cwd(),
+  "test",
+  "fixture-projects",
+  "cli",
+  "telemetry",
+  "analytics",
+  "result.json",
+);
+
+(() => {
```

Comment:
This lambda is not needed.

Reply:
[Done here](https://github.com/NomicFoundation/hardhat/pull/5545/commits/f1ab5a47cabf4d13d1ff3419a96b66e3fec4e2d8)